### PR TITLE
RUBY-2635 Test against server v5.0 + RUBY-2593 pass all server api parameters to getMore & tx operations

### PR DESCRIPTION
**Pull request by comandeo**

### CODE REVIEW

File: .evergreen/config-atlas.yml [hunk 1]
```diff
@@ -396,7 +396,11 @@ axes:
         display_name: "latest"
         variables:
           MONGODB_VERSION: "latest"
-      - id: "4.9"
+      - id: "5.0"
+        display_name: "5.0"
+        variables:
+          MONGODB_VERSION: "4.9"
+      - id: "5.0"
```
Comment (p-mongo) [inline .evergreen/config-atlas.yml:403]: 4.9?

Comment (comandeo) [reply .evergreen/config-atlas.yml]: Oh my 😊
Thanks, fixed

**Comment by p-mongo** [comment]
I am using 4.7 for load balancer testing, can you please put that back for now.

File: .evergreen/config.yml [hunk 2]
```diff
@@ -916,33 +920,22 @@ buildvariants:
     tasks:
       - name: "test-mlaunch"
 
-  - matrix_name: "mongo-4.7"
-    matrix_spec:
-      auth-and-ssl: ["auth-and-ssl", "noauth-and-nossl"]
-      ruby: ["ruby-2.7"]
-      mongodb-version: '4.7'
-      topology: '*'
```
Comment (p-mongo) [inline .evergreen/config.yml:924]: The load-balanced part of this asterisk is the one I am using.

**Comment by p-mongo** [comment]
For the api version required failures  I thought simply getting a correct server release would fix them, but given that this doesn't seem to be the case a separate ticket can be made for investigating and repairing them in the green ci epic.

File: .evergreen/config.yml (hunk 2 above)
Comment (comandeo) [reply .evergreen/config.yml:924]: Added back.

**Review by p-mongo — APPROVED** [review]

**Pull request merged**